### PR TITLE
Adding scripts and tools to run locally

### PR DESCRIPTION
**Pull request by maoo**
This PR depends on https://github.com/symphonyoss/App-Integrations-Commons/pull/27 (to enable `-Prun` maven profile used in `run.sh`)

### CODE REVIEW

**Review by ecarrenho-daitan — APPROVED** [review]
LGTM.

File: local-run/application.yaml.template [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  port: <<KM_PORT>>
+
+key_manager_auth:
+  host: <<KM_AUTH_HOST>>
+  port: <<KM_AUTH_PORT>>
+
+pod_session_manager:
+  host: <<SM_HOST>>
+  port: <<SM_PORT>>
+
```
Comment (robson-sanchez) [inline local-run/application.yaml.template:29]: @maoo We can simplify this template just changing the env.sh script.

This template should contain only the application info and profiles. The API endpoints you could provide on the env.sh, something like that:

```
spring:
  profiles:
    active: <<APP_ID>>

applications:
  <<APP_ID>>:
    state: PROVISIONED
    keystore:
      file: <<CERT_FILE>>
      password: <<CERT_PWD>>
    type: pkcs12
```

File: local-run/env.sh.sample [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+export KM_HOST=foundation-dev.symphony.com
+export KM_PORT=443
+export KM_AUTH_HOST=foundation-dev-api.symphony.com
+export KM_AUTH_PORT=443
+export SM_HOST=foundation-dev-api.symphony.com
+export SM_PORT=443
+
+# Hubspot service account certificate
+export HUBSPOT_CERT_FILE=jira.p12
+export HUBSPOT_CERT_PWD=changeit
```
Comment (robson-sanchez) [inline local-run/env.sh.sample:23]: The environment variables required by the application are:

export INTEGRATION_BRIDGE_DOMAIN=".ngrok.io"
export INTEGRATION_BRIDGE_HOST=myid.ngrok.io
export INTEGRATION_BRIDGE_HTTP_PORT=8080

export POD_HOST=foundation-dev.symphony.com
export POD_PORT=443
export AGENT_HOST=foundation-dev-api.symphony.com
export AGENT_PORT=443
export KEY_MANAGER_HOST=foundation-dev.symphony.com
export KEY_MANAGER_PORT=443
export KEY_MANAGER_AUTH_HOST=foundation-dev-api.symphony.com
export KEY_MANAGER_AUTH_PORT=443
export POD_SESSION_MANAGER_HOST=foundation-dev-api.symphony.com
export POD_SESSION_MANAGER_PORT=443

export APP_ID=hubspot
export CERT_FILE=hubspot.p12
export CERT_PWD=changeit

File: run.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+curl -s https://raw.githubusercontent.com/symphonyoss/contrib-toolbox/master/scripts/inject-vars.sh | bash -s -- ./local-run/application.yaml.template application.yaml
+
+# Cleanup tomcat folder from previous runs
+rm -rf tomcat ; mkdir tomcat
+
+# Run the Spring Boot application
+java -Dlog4j2.outputAllToConsole=true -Dlogs.basedir=target \
+-agentlib:jdwp=transport=dt_socket,server=y,address=5000,suspend=n \
+-jar target/integration.jar \
+--spring.profiles.active=hubspot \
```
Comment (robson-sanchez) [inline run.sh:22]: You could remove the active profiles from the command line arguments as well.

**Pull request merged**